src/cli/changeset/init.ts

```diff
@@ -43,7 +46,7 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
         'No changesets found, attempting to publish any unpublished packages to npm',
       );
 
-      const userNpmrcPath = `${process.env.HOME}/.npmrc`;
+      const userNpmrcPath = `${process.env.HOME as string}/.npmrc`;
```

Comment:
Same here; does Changesets not recognise `NPM_TOKEN` by itself?

Reply:
I'm fairly confident this is because it uses npm publish at some point. We can likely remove this part as we handle the .npmrc creation in Gutenberg